.github/workflows/authors.yml

```diff
@@ -1,5 +1,9 @@
 name: "authors update"
 on:
+  schedule:
+    # Run once a week at 00:05 AM UTC on Sunday.
+    - cron: '5 0 * * 0'
```

Comment:
```suggestion
    - cron: 5 0 * * 0
```

Reply:
Thanks for the suggestion. For people who don't work with YAML all the time, the rules for quotation marks around strings are sufficiently complicated that it makes sense to just quote strings all the time. The arguments against quoting strings all the time tend to be aesthetic (diminishes the "beauty" of YAML) or debatable (reduces the "readability"?!).

And that doesn't even get into the differences between YAML 1.1 (where `yes` without quotes is a boolean) and YAML 1.2 (where `yes` without quotes is a string).

For these reasons, I'm going to leave the string as is.

That said, if we want to enforce consistency, let's get a YAML linter in place. `yamllint` has a [`quoted-strings` rule](https://yamllint.readthedocs.io/en/stable/rules.html#module-yamllint.rules.quoted_strings) that let's you decide you want all strings quoted (and quoted the same way) or that you only want strings quoted when necessary. Once we have enforcement by a tool, I don't care whether the strings are always quoted or only quoted when necessary. Just as long as the tool enforces it.

Reply:
Yes, you are right and I was planning to make a PR for that (by the way I don't know where to put a python dependency in tools). But in the meantime I made #39871 to fix inconsistencies and update some actions.

This would deserve to be in an issue, but for `yamllint` I would have liked to be able to add a plugin to validate github actions, but unfortunately this is not possible, however there is a PR that adds this, but there are these restrictions:
> #. It must be a Python package installable with pip and distributed under
> GPLv3+ same as yamllint.

Reply:
Yeah, I was going to vendor in yamllint but saw that it's GPL, and I don't think we can ship GPL code with this project. (If we can, well, we don't yet.)

So I replicated lint-py a bit and used `pip` to install it. Hopefully that will work. If it lands, once it's in place, then you or I or whoever can try to get the quotes-or-not thing taken care of.